src/zyppinfo.cpp

```diff
@@ -71,14 +72,18 @@ Info ZyppInfo::packageObject( const PoolQuery& q )
 	string status;
 	stringstream format;
 	if ( !installedObject )
-	    status = "Not Installed";
+	    status = i18n( "Not Installed" ).toStdString();
 	else {
-	    if ( updateObject ) {
-		format << "Archaic (version " << installedObject.edition().asString() << " installed)";
-		status = format.str();
+	    if ( updateObject ) { 
+		status = str::Format( "%s %s %s " )
```

Comment:
You should never format translated strings by yourself. Please use i18n( "Archaic (version %s installed)" ).arg( QString::fromStdString(installedObject.edition().asString())) or something along those lines